tests/test_smoke.py

```diff
@@ -563,7 +563,7 @@ def test_tpu_vm():
             f'sky logs {name} 1',  # Ensure the job finished.
             f'sky logs {name} 1 --status',  # Ensure the job succeeded.
             f'sky stop -y {name}',
-            f's=$(sky status --refresh); printf "$s"; echo; echo; printf "$s"  | grep {name} | grep STOPPED',  # Ensure the cluster is STOPPED.
+            f's=$(sky status --refresh); echo "$s\n"; echo; echo; echo "$s\n"  | grep {name} | grep STOPPED',  # Ensure the cluster is STOPPED.
```

Comment:
nit: No need for the extra `\n`? Same for L687, L691, etc in the rest of this file.
```suggestion
            f's=$(sky status --refresh); echo "$s"; echo; echo; echo "$s"  | grep {name} | grep STOPPED',  # Ensure the cluster is STOPPED.
```

Reply:
Done, good catch!